src/cfclient_retrieve.erl

```diff
@@ -0,0 +1,44 @@
+%%%-------------------------------------------------------------------
+%%% @doc
+%%% Pull Feature and Target configuration from the Feature Flags API and store them to an LRU cache.
+%%% @end
+%%%-------------------------------------------------------------------
+-module(cfclient_retrieve).
+
+-export([retrieve_flags/3, retrieve_segments/3]).
+
+-export_type([client_config/0]).
+
+%% TODO - this type should be part of the Client module when it's created. This isn't Golang
+%% where consumers get to say what the interface should be.
+-type client_config() :: {EnvironmentID :: binary(), BearerToken :: binary(), ClusterID :: binary()}.
+
+%% @doc Retrieve all features from the FF API and store them to cache.
+%% @end
+-spec retrieve_flags(Context :: ctx:t(), CachePID :: pid(), ClientConfig :: client_config()) -> ok | not_ok.
+retrieve_flags(Context, CachePID, ClientConfig) ->
+  {BearerToken, EnvironmentID, ClusterID} = ClientConfig,
+  Optional = #{ cfg => #{auth => #{ 'BearerAuth' => <<"Bearer ", BearerToken/binary>>}, host => "https://config.ff.harness.io"},  params => #{cluster => ClusterID }},
```

Comment:
Can you add a comment/note to remove the hardcoding when we get the Config handler done?

Reply:
Done!